package.json

```diff
@@ -1,14 +1,42 @@
 {
   "name": "gridicons",
-  "version": "0.1.0",
+  "version": "1.0.0-alpha.2",
+  "main": "build/index.js",
+  "scripts": {
+    "build": "grunt --verbose",
+    "prepublish": "npm run build"
```

Comment:
Sorry, I mean when the LTS line uses npm 4. As of now, the latest LTS is still on npm 3.10.10